ir/src/main/scala/org/scalajs/core/ir/Printers.scala

```diff
@@ -931,8 +931,42 @@ object Printers {
       printEscapeJS(ident.name, out)
 
     private final def print(propName: PropertyName): Unit = propName match {
-      case lit: StringLiteral => print(lit: Tree)
-      case ident: Ident       => print(ident)
+      case lit: StringLiteral        => print(lit: Tree)
+      case ident: Ident              => print(ident)
+
+      case ComputedName(tree, index) =>
+        print("[")
+        print(tree)
+        print("](")
+        print(index)
+        print(")")
+    }
+
+    private def print(spec: JSNativeLoadSpec): Unit = {
+      def printPath(path: List[String]): Unit = {
+        for (propName <- path) {
+          if (isValidIdentifier(propName)) {
+            print('.')
+            print(propName)
+          } else {
+            print('[')
+            print(propName)
+            print(']')
+          }
+        }
+      }
+
+      spec match {
+        case JSNativeLoadSpec.Global(path) =>
+          print("<global>")
+          printPath(path)
+
+        case JSNativeLoadSpec.Import(module, path) =>
+          print("import(")
+          print(module)
+          print(')')
+          printPath(path)
+      }
```

Comment:
Yes, it was at some point. But it's indeed not relevant anymore. I'll make a separate commit.